docs/run/primitives-get-started.mdx

```diff
@@ -45,6 +46,14 @@ observable = SparsePauliOp("Z" * n_qubits)
 print(f">>> Observable: {observable.paulis}")
 ```
 
+The circuit and observable need to be transformed to only use instructions supported by the system. We'll use the transpiler to do this.
+
+```python
+pm = generate_preset_pass_manager(backend=backend)
+isa_circuit = pm.run(circuit)
```

Comment:
Not that it matters much, but can't we do just

```
isa_circuit = transpile(circuit, backend=backend)
```

Reply:
Consistent with the docs in the transpile section, we are endeavoring to teach use of explicit pass manager instantiation.